pom.xml

```diff
@@ -9,7 +9,7 @@
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <appengine.target.version>1.9.34</appengine.target.version>
+    <appengine.target.version>1.9.36</appengine.target.version>
```

Comment:
Yeah that's what I'm saying!

On Wed, May 4, 2016 at 10:51 AM, Appu Goundan <notifications@github.com>
wrote:

> In pom.xml
> <https://github.com/GoogleCloudPlatform/app-tools-lib-for-java/pull/29#discussion_r62052768>
> :
>
> > @@ -9,7 +9,7 @@
> >
> >    <properties>
> >      <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
> > -    <appengine.target.version>1.9.34</appengine.target.version>
> > +    <appengine.target.version>1.9.36</appengine.target.version>
>
> I would rather we just use whatever is in the user's cloud sdk
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/app-tools-lib-for-java/pull/29/files/958b21d8db741cb08881d4d39cf63c7cc9a12546#r62052768>
>